src/Parser/DeathKnight/Blood/Modules/Abilities.js

```diff
@@ -265,21 +265,30 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.RUNE_1,
         category: Abilities.SPELL_CATEGORIES.HIDDEN,
-        cooldown: haste => 10 / (1 + haste),
+        cooldown: (haste, combatantCurrent) => {
+          const multiplier = combatantCurrent.hasBuff(SPELLS.RUNE_MASTER.id) ? 0.4 : 0;
+          return 10 / (1 + haste) / (1 + multiplier);
```

Comment:
Looks good, my only comment is wondering why you need to update the cooldown in Abilities and then again with SpellUsable.  I'm not as familiar with both files but it seems like if you're updating the CD with SpellUsable that should be enough

Reply:
The reduction I do in SpellsUsable only affects the charge currently recharging. This is needed to ensure that when a rune starts recharging during it starts with the correct cooldown. I could do this by listening to on_updatespellusable and set it there instead but since I use ExpectedCooldown from abilities I thought this would be cleaner. It wouldn't break anything since all cooldowns would be wrong by the same percent, but might be confusing in the future.

Reply:
Gotcha.  It won't be a big deal to update rune cooldowns in the future just gotta remember to update it in two places.  I'm gonna ping for a more experienced reviewer to check this PR just to make everything is good to go